swift/test/Ice/udp/AllTests.swift

```diff
@@ -108,7 +89,9 @@ public func allTests(_ helper: TestHelper) async throws {
         communicator.getProperties().setProperty(key: "Ice.UDP.SndSize", value: "64000")
         seq = ByteSeq(repeating: 0, count: 50000)
         do {
-            replyI.reset()
+            let replyI = PingReplyI()
+            let reply = try replyI.getProxy(adapter)
+
```

Comment:
I find the `catch is Ice.DatagramLimitException {}` below suspicious: either the `setProperty(key: "Ice.UDP.SndSize", value: "64000")` above worked and the invocation should not throw this exception. Otherwise, this test is pointless.

Reply:
I agree seems unnecessary